smother/control.py

```diff
@@ -41,7 +46,7 @@ def start(self):
 
     def save_context(self, label):
         self.data[label] = {
-            key: sorted(map(int, val.keys()))
+            self._normalize_path(key): sorted(map(int, val.keys()))
```

Comment:
so if I'm following so far we are adding extra logic at coverage write time, so that the data files have normalized (potentially relative) paths that can then be interpreted literally at read time. Is that right? Have you considered switching that around, so that writing the reports is "simple" and the extra normalization logic happens when the data file is read?

I'm not sure which would be better. One nice thing about deferring the logic is that you can more easily correct mistakes / refine your path aliases once you've written a bunch of reports -- if path aliases rely on collection-time behavior then any mistakes require re-running the test suite.

OTOH, files are read more often than they are written, so maybe that argues for keeping the logic at write-time